src/inputs/CheckmarksMultipleSelect/CheckmarksMultipleSelect.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React, { useEffect, useState } from 'react';
+import {
+  FormControl,
+  Select,
+  Divider,
+  Checkbox,
+  Button,
+  makeStyles,
+  FormControlLabel,
+  FormGroup,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+
+const useStyles = makeStyles((theme) => ({
+  multipleSelect: {
+    width: '100%',
+  },
+  removeButton: {
+    color: theme.palette.error.main,
+    textTransform: 'none',
+    fontSize: theme.typography.body1.fontSize,
+    display: 'flex',
+    justifyContent: 'left',
+  },
+  menuList: {
+    paddingRight: 0,
+  },
+  menuItem: {
+    marginLeft: 0,
+    marginRight: 0,
+  },
+}));
+export const CheckmarksMultipleSelect = ({
+  labels,
+  roles,
+  agentRoles,
+  onRoleChecked,
+  isRemoveAccessAvailable,
+  removeAccessLabel,
+  onRemoveAccess,
+  isReadOnly,
+}) => {
+  const classes = useStyles();
+  const [grantedRoles, setGrantedRoles] = useState(agentRoles);
+  useEffect(() => {
+    setGrantedRoles(agentRoles);
+  }, [agentRoles]);
+
+  return (
+    <div className={classes.multipleSelect}>
+      <FormControl variant="outlined" fullWidth={true}>
+        <Select
+          multiple
+          disabled={isReadOnly}
+          renderValue={(selected) => selected.map((role) => labels[role]).join(', ')}
```

Comment:
Could we add `role` as a fallback value if `labels[role]` is undefined ? This would display non user-friendly strings, but it's still better than empty strings or "undefined" labels :sweat_smile: